openid-4-verifiable-presentations-1_0.md

```diff
@@ -1421,14 +1303,6 @@ This document also defines the following additional error codes and error descri
 
 - The Wallet does not support any of the formats requested by the Verifier, such as those included in the `vp_formats` registration parameter.
 
-`invalid_presentation_definition_uri`:
-
-- The Presentation Definition URL cannot be reached.
-
-`invalid_presentation_definition_reference`:
-
-- The Presentation Definition URL can be reached, but the specified `presentation_definition` cannot be found at the URL.
-
```

Comment:
I support it, and we never defined `dcql_query_uri` parameter but I think it's worth noting that with this PR, we are losing the ability to pass query language by reference